app/views/refinery/inquiries/inquiry_mailer/notification.text.erb

```diff
@@ -5,6 +5,7 @@
 <%=raw t('.inquiry_starts') %>
 
 <%=raw t('.from') %>: <%= @inquiry.name %>
+<%=raw t('.company') %>: <%= @inquiry.company %>
```

Comment:
this shouldn't appear in the email unless the company field is enabled

Reply:
Right, I added a test which now checks if the show company field setting is on in order to displaying the company.